### PR TITLE
[PyOV] Change default opset9->opset10

### PR DESCRIPTION
**Pull request by akuporos**
### Details:
 - *item1*
 - *...*

### Tickets:
 - *ticket-id*

### CODE REVIEW

**Review by mitruska — APPROVED** [review]
Maintenance of "default opset" value for NodeFactory can be a topic for a separate discussion,
so in general LGTM.

Conclusions:
- In the ops.py files `NodeFactory` is created with opset argument provided directly, so the change of the default value doesn't affect operators:
https://github.com/openvinotoolkit/openvino/blob/8f8e79bb15c2a4bdd91495eeef81960baff17cc3/src/bindings/python/src/openvino/runtime/opset10/ops.py#L20
- Also operators from `opset9` are compatible with `opset10` 
(just fully new operators were added to opset10, no new version of any opset9 operation)

**Review by p-wysocki — APPROVED** [review]

**Review by ilya-lavrenov — APPROVED** [review]

File: src/bindings/python/setup.py [hunk 1]
```diff
@@ -54,6 +54,7 @@
     "openvino.runtime.opset7",
```
Comment (ilya-lavrenov) [inline src/bindings/python/setup.py:54]: better to drop this file at all. It's not a production code, used somewhere in CI only.

**Review by ilyachur — APPROVED** [review]

**Review by jiwaszki — APPROVED** [review]

File: src/bindings/python/setup.py [hunk 2]
```diff
@@ -54,6 +54,7 @@
...
     "openvino.runtime.opset8",
     "openvino.runtime.opset9",
+    "openvino.runtime.opset10",
```
[the remark this answers is not in the document]
Comment (jiwaszki) [reply src/bindings/python/setup.py:57]: @akuporos ^ one additional nitpick from me as well.

**Comment by jiwaszki** [comment]
@ilya-lavrenov port to 2022.3 is still possible?

File: src/bindings/python/setup.py (hunk 1 above)
Comment (akuporos) [reply src/bindings/python/setup.py:54]: yes, we have the task for it -- 81848

**Comment by jiwaszki** [comment]
> * Also operators from `opset9` are compatible with `opset10`
>   (just fully new operators were added to opset10, no new version of any opset9 operation)

As @mitruska stated:
> Also operators from opset9 are compatible with opset10
(just fully new operators were added to opset10, no new version of any opset9 operation)

So I see no conflicts here. All ops should work as usual if imported as something like `import ov.runtime.opsetX as ops`.

**Comment by ilya-lavrenov** [comment]
ok, let's port then

**Pull request merged**